src/components/calcite-action-group/calcite-action-group.e2e.ts

```diff
@@ -1,10 +1,26 @@
 import { accessible, hidden, renders } from "../../tests/commonTests";
+import { newE2EPage } from "@stencil/core/testing";
 
 describe("calcite-action-group", () => {
   it("renders", async () => renders("calcite-action-group"));
 
   it("honors hidden attribute", async () => hidden("calcite-action-group"));
 
+  it("defaults", async () => {
+    const page = await newE2EPage();
+
+    await page.setContent(`
+      <calcite-action-pad>
```

Comment:
To make the test easier to read, can you use only the group component in the test?

Reply:
totes.